src/ol/renderer/canvas/canvastilelayerrenderer.js

```diff
@@ -191,7 +191,12 @@ ol.renderer.canvas.TileLayer.prototype.prepareFrame = function(
 
 
 /**
- * @inheritDoc
+ * @param {ol.Pixel} pixel Pixel.
+ * @param {olx.FrameState} frameState Frame state.
+ * @param {function(this: S, ol.layer.Layer, Object): T} callback Layer callback.
```

Comment:
You could use [ol.Color](https://github.com/openlayers/ol3/blob/master/src/ol/typedefs.js#L102-L109) as type for the pixel value.